src/prepack-cli.js

```diff
@@ -218,19 +224,19 @@ function run(
           break;
         case "debugInFilePath":
           debugInFilePath = args.shift();
-          // do not include this in reproArguments needed by --repro, as debugger behavior is not currently supported for repros
+          // do not include this in reproArguments needed by --repro[OnFatalError/Unconditionally], as debugger behavior is not currently supported for repros
```

Comment:
IIRC, these two parameters are required for debug sessions. Will the `replay` debug session succeed without them?

Reply:
Hmm, I actually believe these fields are unnecessary in general, as the launchRequest handler in the DebugAdapter automatically generates these fields. So the information is required for debugging, but it's not necessary for the user to input them. 

Would you happen to know what situation they're meant for? Otherwise I'd assume they were created to help bootstrap the debugger but are no longer relevant and can be removed.